client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-color-palettes.tsx

```diff
@@ -26,7 +26,10 @@ const ScreenColorPalettes = ( {
 		<>
 			<NavigatorHeader
 				title={ translate( 'Colors' ) }
-				description={ translate( 'Foreground and background colors used throughout your site.' ) }
+				description={ translate(
```

Comment:
Maybe in the future, we can check for `shouldUnlockGlobalStyles` in `<ScreenColorPalettes/>` and `<ScreenFontPairings/>` in order to include the upsell message. I don't think it's necessary to include it now since the PA is only available in the onboarding flow 🙂